### ISSUE TITLE
Why are there both .prettierrc and tools/prettier.config.js? 

### ISSUE THREAD

**Issue by jjangga0214** [andrewthauer/ts-monorepo#1]
Hi, 

why are there both .prettierrc and tools/prettier.config.js? 

Thanks for the repo by the way!

**Comment by andrewthauer** [comment]
Yeah, that's probably not ideal. It would definitely make sense to consolidate these. The original rationale was to have the prettier config extendable by each package, where as the root one would handle non `package` files. Prettier rules should ideally be consistent, but ignoring / overriding some specific file types might be useful in some packages.
